zio-sql/jvm/src/main/scala/Sql.scala

```diff
@@ -627,16 +627,24 @@ trait Sql {
   }
 
   object FunctionDef {
-    //match functions
+    //math functions
     val Abs   = FunctionDef[Double, Double](FunctionName("abs"))
+    val Acos  = FunctionDef[Double, Double](FunctionName("acos"))
+    val Asin  = FunctionDef[Double, Double](FunctionName("asin"))
+    val Atan  = FunctionDef[Double, Double](FunctionName("atan"))
     val Ceil  = FunctionDef[Double, Double](FunctionName("ceil"))
+    val Cos   = FunctionDef[Double, Double](FunctionName("cos"))
     val Exp   = FunctionDef[Double, Double](FunctionName("exp"))
     val Floor = FunctionDef[Double, Double](FunctionName("floor"))
     //val Log = FunctionDef[Double, Double](FunctionName("log")) //not part of SQL 2011 spec
```

Comment:
`not part of SQL 2011 spec` this is not the only problem with `log` function:
Mysql `log` returns the natural logarithm but Postgresql base10 logarithm